crates/sncast/src/starknet_commands/account/create.rs

```diff
@@ -103,19 +103,21 @@ pub async fn create(
         add_created_profile_to_configuration(&add_profile, &config, &None)?;
     }
 
+    let profile_message = if add_profile.is_some() {
+        format!(
+            "Profile {} successfully added or updated in snfoundry.toml",
+            add_profile.clone().expect("Failed to get profile name")
+        )
+    } else {
+        "No profile added or updated in snfoundry.toml".to_string()
+    };
+
```

Comment:
Let's revert back to have this in the previous place

Reply:
I have reverted this in the new commit